app/application/template.hbs

```diff
@@ -1,6 +1,6 @@
-{{outlet}}
-
 <EmberNotify @messageStyle="bootstrap" />
 <SyModalTarget />
 
 <div id="filter-sidebar-target"></div>
+
+{{outlet}}
```

Comment:
Just for the future to keep in mind if we have funky rendering issues, (same as in wigl) this might be the cause